bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -76,6 +72,16 @@ pub use pallet::*;
 
 pub const LOG_TARGET: &str = "snowbridge-inbound-queue";
 
+pub trait RewardProcessor<T: frame_system::Config> {
+	fn process_reward(who: T::AccountId, message: Message) -> DispatchResult;
```

Comment:
we need the channel here as well (to keep the previous format)

Reply:
Since you already have channel id available at the place where you are calling this, it's better if we take that instead of fetching it again from channel id lookup. 

Because otherwise we are duplicating the logic of fetching channel id and if the two goes out of sync for whatever reason we might have a problem.